plugins/services/src/js/components/modals/NewCreateServiceModalForm.js

```diff
@@ -233,6 +234,16 @@ class NewCreateServiceModalForm extends Component {
     this.setState({batch, appConfig: this.getAppConfig(batch)});
   }
 
+  handleSetItem({value, path}) {
```

Comment:
I mean `setItem` is being used solely for setting an item :D In other cases we set an empty item in a reducer directly. Like service endpoints or volumes. I don't understand why can't we use the same approach here?